alfresco-docker-transform-misc/src/main/java/org/alfresco/transformer/MiscController.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * #%L
+ * Alfresco Transform Core
+ * %%
+ * Copyright (C) 2005 - 2019 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * -

```

Comment:
Again these license headers look wrong for community code. Add to the ATS issue or change to be the same as the other T-Engines.